packages/db/src/Breakdown/MaterialUsageBreakdown.tsx

```diff
@@ -0,0 +1,107 @@
+import {Servant, Region} from "@atlasacademy/api-connector";
+import {AxiosError} from "axios";
+import React from "react";
+import {Table} from "react-bootstrap";
+import ErrorStatus from "../Component/ErrorStatus";
+import Loading from "../Component/Loading";
+import {Link} from "react-router-dom";
+import FaceIcon from "../Component/FaceIcon";
+import {MaterialUsageData} from "../Page/Material/MaterialUsageData";
+import Api from "../Api";
+import Manager from "../Setting/Manager";
+
+interface IProps {
+    region: Region;
+    usageData: MaterialUsageData[];
+}
+
+interface IState {
+    error?: AxiosError;
+    loading: boolean;
+    servants: Servant.ServantBasic[];
```

Comment:
@Mitsunee @minhducsun2002 Does this mean the page loads both basic and nice? basic is pretty light but probably best to only load nice.

Reply:
the material page uses nice_servant and the breakdown uses basic to grab whichever default face that has because TypeScript was complaining that extraAssets from nice_servant may possibly be undefined.

Reply:
basically I assumed that basic_servant is probably cached anyways, since the only 2 ways to get to a material page is either a servant page or the materials page

Reply:
If that's the only use for basic, let's fix the TS error instead. You can check for existence of extraAssets: `if servant.extraAssets { ... }` or `servant.extraAssets?` I think.

Another thing, I don't think the materials page would load basic_servant?

Reply:
I'll locally try using `nice_servant.json` for the breakdown again and post on discord.

materials only loads `nice_item.json`. Material (the one this refers to?) requires `nice_servant.json` for the ascenion/skill/costumeMaterial data.